Wrote a common service for methods that will be used in both booking and cancellation; has 100% unit test coverage

justin-obrien-smoothstack opened the pull request.

justin-obrien-smoothstack committed:
Move method to check if user is traveler into separate service class

Both the booking and cancellation processes will use this method, so it
makes sense to have it in its own service

trevorhuis-smoothstack commented on src/main/java/com/ss/training/utopia/counter/service/UserService.java:
```diff
@@ -0,0 +1,26 @@
+
+	@Autowired
+	UserDao userDao;
+
```
great idea to be able to check for this.

justin-obrien-smoothstack committed:
Merge branch 'dev' into service-user

justin-obrien-smoothstack committed:
Add service method to get user by username

justin-obrien-smoothstack committed:
Write 100% test of service method that gets user by username

justin-obrien-smoothstack commented:
Added another method and test.

justin-obrien-smoothstack committed:
Modify user DAO formatting

justin-obrien-smoothstack committed:
Merge branch 'dao-user' into service-user

justin-obrien-smoothstack committed:
Remove DAO exception handling from user service; update tests

justin-obrien-smoothstack commented:
Removed DAO exception catching.

justin-obrien-smoothstack committed:
Renamed UserService to CommonService

justin-obrien-smoothstack committed:
Added method to get all airports to common service

Previously it was in booking service, but it will be used during
cancellation too

justin-obrien-smoothstack commented:
Renamed from UserService to CommonService and added method to get all airports.

justin-obrien-smoothstack committed:
Add Service annotation to common service
